habitat-hitl/habitat_hitl/environment/controllers/gui_controller.py

```diff
@@ -56,82 +134,112 @@ def act(self, obs, env):
         KeyNS = GuiInput.KeyNS
         gui_input = self._gui_input
 
+        # Note addition of 180 degrees due to our camera convention
+        turn_angle = self.angle_from_sim_obj_foward_dir_to_target_yaw(
+            self._articulated_agent.sim_obj,
+            self._cam_yaw + float(mn.Rad(mn.Deg(180))),
+        )
```

Comment:
>180 degrees due to our camera convention

I may be confronted to a math error due to this. Could you provide some context on why the 180 degree rotation brings us to the desired result here?

Reply:
Added a comment in code. This is unrelated to sensor conventions and strictly due to camera_helper.py implementation details.